services/api/service.go

```diff
@@ -394,6 +404,16 @@ func (api *RelayAPI) StartServer() (err error) {
 		}
 	}()
 
+	// Start regular payload attributes updates
+	go func() {
```

Comment:
should the subscription only be started if `ffEnableSSEPayloadAttributes` is true?